### PR TITLE
Fixed the changes to jenkins run file

### PR DESCRIPTION
**Pull request by athiruma**

### CODE REVIEW

**Review by ebattat — APPROVED** [review]
/LGTM

File: jenkins/clouds/aws/daily/org_cost_explorer/run_org_upload_es.py [hunk 1]
```diff
@@ -18,9 +18,9 @@
...
 combine_vars = lambda item: f'{item[0]}="{item[1]}"'
 
 common_input_vars['es_index'] = 'cloud-governance-payer-billing-reports'
 common_envs = list(map(combine_vars, common_input_vars.items()))
-os.system(f"""podman run --rm --name cloud-governance -e policy="cost_explorer_payer_billings" -e AWS_ACCOUNT_ROLE="{AWS_ACCOUNT_ROLE}" -e account="PERF-DEPT" -e AWS_ACCESS_KEY_ID="{AWS_ACCESS_KEY_ID_DELETE_PERF} -e AWS_SECRET_ACCESS_KEY="{AWS_SECRET_ACCESS_KEY_DELETE_PERF}" -e SPREADSHEET_ID="{COST_SPREADSHEET_ID}" -e {' -e '.join(common_envs)} -e {' -e '.join(common_envs)} -v "{GOOGLE_APPLICATION_CREDENTIALS}":"{GOOGLE_APPLICATION_CREDENTIALS}" quay.io/ebattat/cloud-governance:latest""")
+print(f"""podman run --rm --name cloud-governance -e policy="cost_explorer_payer_billings" -e AWS_ACCOUNT_ROLE="{AWS_ACCOUNT_ROLE}" -e account="PERF-DEPT" -e AWS_ACCESS_KEY_ID="{AWS_ACCESS_KEY_ID_DELETE_PERF}" -e AWS_SECRET_ACCESS_KEY="{AWS_SECRET_ACCESS_KEY_DELETE_PERF}" -e SPREADSHEET_ID="{COST_SPREADSHEET_ID}" -e {' -e '.join(common_envs)} -v "{GOOGLE_APPLICATION_CREDENTIALS}":"{GOOGLE_APPLICATION_CREDENTIALS}" quay.io/ebattat/cloud-governance:latest""")
```
Comment (ebattat) [inline jenkins/clouds/aws/daily/org_cost_explorer/run_org_upload_es.py:26]: Are you sure print instead of os.system ?

Comment (athiruma) [reply jenkins/clouds/aws/daily/org_cost_explorer/run_org_upload_es.py]: No, 
I tested whether the statement was working or not. I forgot to delete the print.
I reverted the print to os.system.

File: jenkins/clouds/aws/daily/org_cost_explorer/run_org_upload_es.py [hunk 2]
```diff
@@ -18,9 +18,9 @@
...
 combine_vars = lambda item: f'{item[0]}="{item[1]}"'
 
 common_input_vars['es_index'] = 'cloud-governance-payer-billing-reports'
 common_envs = list(map(combine_vars, common_input_vars.items()))
-os.system(f"""podman run --rm --name cloud-governance -e policy="cost_explorer_payer_billings" -e AWS_ACCOUNT_ROLE="{AWS_ACCOUNT_ROLE}" -e account="PERF-DEPT" -e AWS_ACCESS_KEY_ID="{AWS_ACCESS_KEY_ID_DELETE_PERF} -e AWS_SECRET_ACCESS_KEY="{AWS_SECRET_ACCESS_KEY_DELETE_PERF}" -e SPREADSHEET_ID="{COST_SPREADSHEET_ID}" -e {' -e '.join(common_envs)} -e {' -e '.join(common_envs)} -v "{GOOGLE_APPLICATION_CREDENTIALS}":"{GOOGLE_APPLICATION_CREDENTIALS}" quay.io/ebattat/cloud-governance:latest""")
+os.system(f"""podman run --rm --name cloud-governance -e policy="cost_explorer_payer_billings" -e AWS_ACCOUNT_ROLE="{AWS_ACCOUNT_ROLE}" -e account="PERF-DEPT" -e AWS_ACCESS_KEY_ID="{AWS_ACCESS_KEY_ID_DELETE_PERF}" -e AWS_SECRET_ACCESS_KEY="{AWS_SECRET_ACCESS_KEY_DELETE_PERF}" -e SPREADSHEET_ID="{COST_SPREADSHEET_ID}" -e {' -e '.join(common_envs)} -v "{GOOGLE_APPLICATION_CREDENTIALS}":"{GOOGLE_APPLICATION_CREDENTIALS}" quay.io/ebattat/cloud-governance:latest""")
```
Comment (ebattat) [inline jenkins/clouds/aws/daily/org_cost_explorer/run_org_upload_es.py:26]: -e {' -e '.join(common_envs)}
What it means common env ?

Comment (athiruma) [reply jenkins/clouds/aws/daily/org_cost_explorer/run_org_upload_es.py:26]: I put the common vars in one variable and add to the command.
join add the -e after the first iter, so added one *-e* before the joining

Comment (ebattat) [reply jenkins/clouds/aws/daily/org_cost_explorer/run_org_upload_es.py:26]: Great initiative, but we need to document somewhere what are the common env

Comment (athiruma) [reply jenkins/clouds/aws/daily/org_cost_explorer/run_org_upload_es.py:26]: Above the command declared the common_envs

Comment (ebattat) [reply jenkins/clouds/aws/daily/org_cost_explorer/run_org_upload_es.py:26]: Ok

**Pull request merged**